### DISCUSSION TITLE
Transparencies are enabled but not working

### DISCUSSION THREAD

**Discussion by InvaleGO** [Atlas-OS/Atlas#1308] [Help & Support]
Hello, good afternoon.
In the settings, transparencies are enabled, but they do not work.
I tried uninstalling and reinstalling the drivers, but they still do not work.
How can I fix this?
Thanks.

**Comment by AudiEtoffe** [comment]
Are you certain your video adapter drivers are installed and working? Can you access their control panel?

**Comment by SpyderDoesStuff** [comment]
i had this problem as well. but that was because the drivers weren't installed. after installing them transparency worked. i would try installing the drivers for your graphics card and see if that fixes it
